microshift_backup_and_restore/microshift-auto-recover-manual-backup.adoc

```diff
@@ -0,0 +1,30 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="microshift-auto-recover-manual-backup"]
+include::_attributes/attributes-microshift.adoc[]
+= Auto recovery from manual backups
+:context: microshift-backup-and-restore
+
+toc::[]
+
+Auto-recovery uses the manual backup to automatically restore backups when {microshift-short} fails to start.
+
+You can use the following command-line options with the existing `backup` and `restore` commands in this feature:
+
+[NOTE]
+====
+* `--dont-save-failed` option can be used only with the `restore` command.
+* `--auto-recovery` option can be used with both the`backup` and `restore` commands.
+====
+
+* --auto-recovery: This option changes the interpretation of the `PATH` argument from the final backup path to a directory holding all the auto-recovery backups.
+* --dont-save-failed: This option disables backing up of failed {microshift-short} data.
```

Comment:
This statement is very clear. Can it be re-worded ?

Reply:
done